### DISCUSSION TITLE
poverty

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#31] [Announcements]
# poverty

A case for giving more money to those who are bad with money.

https://nonzerosum.games/poverty.html

**Comment by MattBallAZ** [comment]
This is great.
Robert Sapolsky's work on this (e.g., early stress' impact on the brain) has been so important. Starting with Why Zebras Don't Get Ulcers, to Behave, even to Determined (the best of the three overall, but not as focused on stress) he has done incredible work. The more people understand those insights, the better our society could be.
Or, you know, not.   :-P
